test/db/anal/arm

```diff
@@ -788,27 +788,28 @@ aa
 afv~var
 EOF
 EXPECT=<<EOF
-var int16_t var_54h @ sp+0x0
```

Comment:
i think i'm missing why vars at `sp+0x0` were named `var_54h` :/ Do you know why?

Reply:
see https://github.com/radareorg/radare2/pull/17255#discussion_r456090977